config/routes.rb

```diff
@@ -11,4 +11,8 @@
   # root "posts#index"
   root "standings#index"
   get "/matches", to: "matches#index"
+  get "/simulations", to: "simulations#index"
+  post "/simulations", to: "simulations#create"
+  get "/simulations/new", to: "simulations#new", as: :new_simulation
+  get "/simulations/:id", to: "simulations#show", as: :simulation
```

Comment:
Rails convention would be to treat these as resources, and brings your route definition to one line:

```suggestion
  resources :simulations, only: %i[index create new show]
```